### PR TITLE
Wip/readme

### PR DESCRIPTION
**Pull request by csantosbh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,109 +37,95 @@ class, but can also be customized to work with any arbitrary data structure.
 * Auto-load buffers being visualized in the previous debug session
 * Designed to scale well for HighDPI displays
```
Comment (csantosbh) [inline README.md]: I guess I could also add an icon indicating that it is now multiplatform (linux and mac) :)

File: README.md [hunk 2]
```diff
@@ -185,57 +171,21 @@ QMAKE_LFLAGS += \
...
-    python3 /path/to/gdb-imagewatch/gdb-imagewatch.py --test
-
-If the installation was succesful, you should see the GDB ImageWatch window
-with the same `sample_buffer_1` and `sample_buffer_2` buffers from the image on
-the header of this page.
-
 ## Using plugin
 
-When GDB hits a breakpoint, the GDB ImageWatch window will be opened. You only
+When GDB hits a breakpoint, the Open Image Debugger window will be opened. You only
```
Comment (brunoalr) [inline README.md]:
```suggestion
When the debugger hits a breakpoint, the Open Image Debugger window will be opened. You only
```

File: README.md (hunk 1 above)
Comment (brunoalr) [reply README.md]: which kind of icon? xD

File: README.md [hunk 3]
```diff
@@ -37,109 +37,95 @@ class, but can also be customized to work with any arbitrary data structure.
...
-For that, run the `gdb` command, and inside the gdb console, run the following
-command:
+### Building the Open Image Debugger
+Clone the source code to any folder you prefer and initialize the
+submodules:
 
-```gdb
-(gdb) python import sys; print(sys.version)
+```shell
+git clone git@github.com:OpenImageDebugger/OpenImageDebugger.git
```
Comment (brunoalr) [inline README.md]:
```suggestion
git clone https://github.com:OpenImageDebugger/OpenImageDebugger.git
```

File: README.md [hunk 4]
```diff
@@ -185,57 +171,21 @@ QMAKE_LFLAGS += \
...
-
 ## Using plugin
 
-When GDB hits a breakpoint, the GDB ImageWatch window will be opened. You only
+When GDB hits a breakpoint, the Open Image Debugger window will be opened. You only
 need to type the name of the buffer to be watched in the "add symbols" input,
 and press `<enter>`.
 
-Alternatively, you can also invoke the GDB ImageWatch window directly from GDB
+Alternatively, you can also invoke the Open Image Debugger window directly from GDB
```
Comment (brunoalr) [inline README.md]: How does it work for LLDB?

File: README.md [hunk 5]
```diff
@@ -289,12 +239,12 @@ folder to Octave/Matlab `path` variable and call
 If you're not using gdb from the command line, make sure that your IDE is
```
Comment (brunoalr) [inline README.md]:
```suggestion
If you're not using the debugger from the command line, make sure that your IDE is
```

File: README.md (hunk 1 above)
Comment (csantosbh) [reply README.md]: Better to just leave the OS names in the features list and add icons later when the user can download binaries in the future.

File: README.md (hunk 4 above)
Comment (csantosbh) [reply README.md]: This is a legacy instruction; now OID starts up automatically after the first debugger stop. It has been like this for a while now, I just never remembered to get rid of this instruction. Doing it now...

File: README.md (hunk 5 above)
Comment (csantosbh) [reply README.md]: Another legacy section of README. Getting rid of this.

**Pull request merged**